snapdtool/cmdutil.go

```diff
@@ -112,9 +112,32 @@ func CommandFromSystemSnap(name string, cmdArgs ...string) (*exec.Cmd, error) {
 	cmdPath := filepath.Join(root, name)
 
 	if from == "snapd" {
-		// We do not need to specify the linker or library
```

Comment:
^^^ "Only files from core need this hack" <- statement in func comment not true anymore.